RIGImageGallery/RIGSingleImageViewController.swift

```diff
@@ -8,34 +8,45 @@
 
 import UIKit
 
-class RIGSingleImageViewController: UIViewController {
+open class RIGSingleImageViewController: UIViewController {
 
-    var viewerItem: RIGImageGalleryItem? {
+    open var viewerItem: RIGImageGalleryItem? {
         didSet {
             viewerItemUpdated()
         }
     }
 
-    let scrollView = RIGAutoCenteringScrollView()
+    open let scrollView = RIGAutoCenteringScrollView()
+    open let activityIndicator: UIActivityIndicatorView = {
+        let activityIndicator = UIActivityIndicatorView()
+        activityIndicator.activityIndicatorViewStyle = .gray
+        activityIndicator.hidesWhenStopped = true
+        return activityIndicator
+    }()
 
-    convenience init(viewerItem: RIGImageGalleryItem) {
+    public convenience init(viewerItem: RIGImageGalleryItem) {
         self.init()
         self.viewerItem = viewerItem
         viewerItemUpdated()
     }
 
-    override func loadView() {
+    open override func loadView() {
```

Comment:
Since it looks like people will want to subclass this if they want more functionality on single pages, I'd add documentation to this function to explain that they probably want to call `super` here.